lib/modules/versioning/ubuntu/index.ts

```diff
@@ -1,18 +1,16 @@
 import { regEx } from '../../../util/regex';
+import { DistroInfo } from '../distro';
 import type { NewValueConfig, VersioningApi } from '../types';
-import {
-  getCodenameByVersion,
-  getVersionByCodename,
-  isCodename,
-} from './distribution';
 
 export const id = 'ubuntu';
 export const displayName = 'Ubuntu';
-export const urls = ['https://changelogs.ubuntu.com/meta-release'];
+export const urls = [
+  'https://changelogs.ubuntu.com/meta-release',
+  "https://debian.pages.debian.net/distro-info-data/ubuntu.csv'",
```

Comment:
```suggestion
  'https://debian.pages.debian.net/distro-info-data/ubuntu.csv',
```

Reply:
done

Reply:
please read https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting